app/src/main/kotlin/org/imperial/mrc/hint/controllers/BaselineController.kt

```diff
@@ -0,0 +1,60 @@
+package org.imperial.mrc.hint.controllers
+
+import org.apache.tomcat.util.http.fileupload.FileUtils
+import org.imperial.mrc.hint.AppProperties
+import org.pac4j.core.config.Config
+import org.pac4j.core.context.WebContext
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.multipart.MultipartFile
+import java.io.File
+
+@RestController
+@RequestMapping("/baseline")
+class BaselineController(val context: WebContext,
+                         val pac4jConfig: Config,
+                         val appProperties: AppProperties) {
+
+    @PostMapping("/pjnz/upload")
```

Comment:
Doesn't really matter but I suppose the /upload isn't really needed as it's implied by a POST to a pjnz endpoint?

Reply:
Good point, nicer to be RESTful where we can!